src/modules/shared/components/MediaCard/MediaCard.module.scss

```diff
@@ -29,7 +29,7 @@ $c-media-card-no-video-intended-spacing: 0.9rem;
 
 				> img {
 					position: static !important;
-					height: auto !important;
+					height: calc(238px / 16 * 9) !important;
```

Comment:
`238px` in een variabele steken en toepassen waar nodig + responsiviteit is niet in orde.
![image](https://user-images.githubusercontent.com/90180417/166647623-4536f55a-8043-4854-b80a-0c680ad84252.png)
![image](https://user-images.githubusercontent.com/90180417/166647639-11fa7b21-eb97-4ae8-9835-ee374cd9ac11.png)
![image](https://user-images.githubusercontent.com/90180417/166647648-57ae92be-2de4-4d19-bae5-4c38e03f3b64.png)
Ik zou deze PR gewoon close'n tbh, de moeite niet waard.